Android rpi

hskang9 opened the pull request:
Android로 raspberry pi를 제어하는 프로젝트입니다.

hskang9 committed:
add android application

hskang9 committed:
IoT device for RpiController

toori67 commented on Projects/LED_Tutorial/rpi-android/RpiController/app/src/main/java/android/iotlabs/com/rpicontroller/MainActivity.java:
```diff
@@ -0,0 +1,108 @@
+public class MainActivity extends Activity implements View.OnClickListener {
+    TextView message;
+    Button btn_connect, btn_on, btn_off, btn_close;
+    Button c4, d4, e4, f4, g4, a4, b4, c5;
```
변수명이 조금더 명확 했으면 좋겠습니다.

hskang9 replied:
수정하겠습니다.

toori67 commented on Projects/LED_Tutorial/rpi-android/RpiController/app/src/main/java/android/iotlabs/com/rpicontroller/MainActivity.java:
```diff
@@ -0,0 +1,108 @@
+
+public class MainActivity extends Activity implements View.OnClickListener {
+    TextView message;
+    Button btn_connect, btn_on, btn_off, btn_close;
```
변수명은 자바에서 camelCase 로 써주세요!
hungarian_style (x) 
camelCase(o)

hskang9 replied:
알겠습니다.

toori67 reviewed (changes requested):
리뷰사항 수정 부탁드려요 :)

hskang9 committed:
리뷰 보고 수정
1. MainActivity.java: hungarian_style -> camelCase
2. MainActivity.java: 변수명은 'btn(계이름)'으로 바꾸고 주석 추가

toori67 reviewed (approved):
:+1:
